src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -16,7 +16,7 @@
 
 from collections import namedtuple, OrderedDict
 from contextlib import contextmanager
-from functools import wraps
+from functools import wraps, reduce, update_wrapper
```

Comment:
nit, `reduce` is not necessary here. neither should it be a part of the backport.